src/praisonai-code/praisonai_code/cli/ui/__init__.py

```
@@ -0,0 +1,162 @@
+        _lazy_cache[name] = MiddleGroundBackend
+        return MiddleGroundBackend
+    
+    if name == 'select_backend':
+        from .selector import select_backend
+        _lazy_cache[name] = select_backend
+        return select_backend
+    
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+
+def select_backend(config: Optional['UIConfig'] = None):
```

Ruff (0.15.20)

[error] Undefined name `UIConfig`

(F821)